frontend/public/components/edit-yaml.jsx

```diff
@@ -552,6 +556,41 @@ export const EditYAML = connect(stateToProps)(
       return yamlService;
     }
 
+    generateSidebarLinkLabel(showSchema, showSamples, showSnippets) {
+      if (showSchema && showSamples && showSnippets) {
```

Comment:
I would probably just say "Show Sidebar" at this point and skip this. Since it's visible by default and they chose to hide it, users already knows what's there.